openedx/features/edly/management/commands/populate_cohorts_for_course.py

```diff
@@ -0,0 +1,49 @@
+"""
+Populate cohorts for a course if cohorts tab is broken.
+"""
+
+import logging
+
+from django.core.management import BaseCommand, CommandError
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.courseware.courses import get_course
+from openedx.core.djangoapps.course_groups.models import CourseCohort, CourseUserGroup
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Populate cohorts for a course'
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            '--course',
+            action='store',
+            type=str,
+            required=True,
+            help='The course ID of the course whose cohorts need to be populated.'
+        )
+
+    def handle(self, *args, **options):
+        course_id = options['course']
+        try:
+            course_key = CourseKey.from_string(course_id)
+        except InvalidKeyError:
+            logger.exception('Course ID %s is incorrect', course_id)
```

Comment:
No need to add explicit log here since we are raising `CommandError` below

Reply:
removed.